app/views/RoomsListView/ListHeader/index.js

```diff
@@ -1,28 +1,80 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import Sort from './Sort';
-import Encryption from './Encryption';
+import { withTheme } from '../../../theme';
+import I18n from '../../../i18n';
+import * as List from '../../../containers/List';
+import { E2E_BANNER_TYPE } from '../../../lib/encryption/constants';
+import { CustomIcon } from '../../../lib/Icons';
+import { themes } from '../../../constants/colors';
 
 import OmnichannelStatus from '../../../ee/omnichannel/containers/OmnichannelStatus';
 
-const ListHeader = React.memo(({
-	searching,
-	sortBy,
-	toggleSort,
-	goEncryption,
-	goQueue,
-	queueSize,
-	inquiryEnabled,
-	encryptionBanner,
-	user
-}) => (
-	<>
-		<Encryption searching={searching} goEncryption={goEncryption} encryptionBanner={encryptionBanner} />
-		<Sort searching={searching} sortBy={sortBy} toggleSort={toggleSort} />
-		<OmnichannelStatus searching={searching} goQueue={goQueue} inquiryEnabled={inquiryEnabled} queueSize={queueSize} user={user} />
-	</>
-));
+const ListHeader = React.memo(
```

Comment:
Same as the other component.
Review your IDE and change only affected lines, please.